src/lib/actions.ts

```diff
@@ -553,7 +553,8 @@ export async function fetchPassageContent(studyId: string) {
           stropheStylingMap.set(obj.stropheId, { borderColor: obj.borderColor, colorFill: obj.colorFill, expanded: obj.expanded });
         });
 
-        const passageContent = await xataClient.db.heb_bible
+        // const passageContent = await xataClient.db.heb_bible
+        const passageContent = await xataClient.db.heb_bible_Oct31_2024
```

Comment:
Once this is approved by Toby, switch the name of heb_bible to heb_bible_old and replace this name with heb_bible.